cluster/gce/trusty/configure.sh

```diff
@@ -144,8 +144,17 @@ install_additional_packages() {
   fi
 }
 
-# Downloads kubernetes binaries and salt tarball, unpacks them, and places them
-# to suitable directories.
+# Retry a download until we get it.
+#
+# $1 is the file to creat
+# $2 is the URL to download
+download_or_bust() {
```

Comment:
The [version in configure-vm.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/configure-vm.sh#L110) uses an until to actually keep trying. The comment here says it retries but it doesn't seem as though that is implemented.